Makefile

```diff
@@ -35,6 +39,61 @@ start: ## Start the development environment (use Docker)
 stop: ## Stop FreshRSS container if any
 	docker stop freshrss-dev
 
+##########
+## I18N ##
+##########
+.PHONY: i18n-format
+i18n-format: ## Format I18N files
+	@$(PHP) ./cli/manipulate.translation.php -a format
+	@echo Files formatted.
+
+.PHONY: i18n-add-language
+i18n-add-language: ## Add a new supported language
+ifndef lang
+	@echo To add a new language, you need to provide one in the "lang" variable.
+	@exit 10
+endif
+	@$(PHP) ./cli/manipulate.translation.php -a add -l $(lang)
+	@echo Language added.
+
+.PHONY: i18n-add-key
+i18n-add-key: ## Add a key
```

Comment:
I wonder if we should be more explicit, something like "Add a I18N key to all languages" (maybe i18n is too much)